foreman_rh_cloud.gemspec

```diff
@@ -22,7 +22,7 @@ Gem::Specification.new do |s|
 
   s.add_dependency 'foreman_ansible'
   s.add_dependency 'foreman-tasks'
-  s.add_dependency 'katello'
+  s.add_runtime_dependency 'katello', '>= 4.14.0.rc1.1', '< 4.15.0'
```

Comment:
Why do we need to add the upper limit? Are we sure that the code won't work for Katello 4.15?

Reply:
I thought that was just the custom. I don't see a reason we can't remove it.

Reply:
Removed the upper limit.